R/convGDX2MIF.R

```diff
@@ -96,9 +96,12 @@ convGDX2MIF <- function(gdx,
 
 .findInconsistenSetElements <- function(brickSets, gdx) {
   m <- Container$new(gdx)
-  setsGdx <- setNames(m$getSymbols(names(brickSets)), names(brickSets))
+  sets <- unique(.split(names(brickSets)))
```

Comment:
As for the comment on ```.split``` in ```reportBrickSet.R```: It is hard to find the definition of ```.split``` (as well as ```.combinations``` below) as they are not found in the file (which is what I would usually expect). But as mentioned below, I'm also unsure what's the best way to handle this.

Reply:
moved to helper.R